src/display/display_utils.js

```diff
@@ -635,6 +635,67 @@ class PDFDateString {
   }
 }
 
+function makeColorComp(n) {
+  return Math.floor(Math.max(0, Math.min(1, n)) * 255)
+    .toString(16)
+    .padStart(2, "0");
+}
+
+const ColorConverters = {
+  // PDF specifications section 10.3
+  CMYK_G(components) {
+    const [c, y, m, k] = components;
```

Comment:
Here, and everywhere below, is there a reason for not using the following (more compact) pattern instead?
```suggestion
  CMYK_G([c, y, m, k]) {
```

Reply:
Thx and fixed.